.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Slow Tests
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  schedule:
+    - cron: "0 1 * * 1"
+  workflow_dispatch:
+
+env:
+  RUST_TEST_THREADS: 4
+
+jobs:
+  build:
+    runs-on: [self-hosted, X64]
+    container:
+      image: ghcr.io/espressosystems/nix:main
+      volumes:
+        - github_nix:/nix
+    steps:
+      - uses: styfle/cancel-workflow-action@0.9.1
+        name: Cancel Outdated Builds
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: cachix/cachix-action@v10
+        with:
+          name: espresso-systems-private
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
+
+      - name: Potential broken submodules fix
+        run: |
+          git checkout -f $(git -c user.name=x -c user.email=x@x commit-tree $(git hash-object -t tree /dev/null) < /dev/null) || :
+      - uses: actions/checkout@v2
+        name: Checkout Repository
+
+      - name: Initialize Nix Shell
+        run: nix-shell --run "echo Init"
+
+      - name: Cache cargo
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry/index
+            ~/.cargo/registry/cache
+            ~/.cargo/git
+            target
+          # todo: add nix key, for example:  nix-instantiate shell.nix | sha256sum  | head -c 10
+          key: cape-v5-${{ hashFiles('Cargo.lock') }}
+
+      - name: Run Tests
+        run: cargo test --release --features=slow-tests
```

Comment:
Since this workflow is using the Nix build, I think you need `nix-shell --run "cargo test --release --features=slow-tests"`. Otherwise follow https://github.com/EspressoSystems/espresso/pull/392 to remove the Nix stuff.